docker/docker-build.bat

```diff
@@ -5,12 +5,17 @@ echo.
 set "DOCKERUSER=globalhealthyliveablecities"
 set "PACKAGE=global-indicators"
 for /f "usebackq" %%x in ("%CD%\..\.ghsci_version") do set VERSION=%%x
+echo %PACKAGE% version %VERSION%
 :: login and remove any existing containers or images
 docker login
 
-:: build the image and export the conda env to yml
+:: build test image and export the conda env to yml
 docker build -t %DOCKERUSER%/%PACKAGE% .
-docker run --rm -it --shm-size=2g --net=host -v "%CD%":/home/ghsci globalhealthyliveablecities/global-indicators /bin/bash -c "pip list --format=freeze > ./requirements.txt"
+docker run --rm -it --shm-size=2g --net=host -v "%CD%":/home/ghsci %DOCKERUSER%/%PACKAGE% /bin/bash -c "pip list --format=freeze > ./requirements.txt"
```

Comment:
Why set `--shm-size` here?

Reply:
I think this is because this 'docker run' command is launching the container using a historical snippet of code, overriding the deafult size that was (or perhaps was?) 64mb, since that would not be sufficient to adequately/reliably analyse most cities.     However --- good point that in that particular context, setting shm-size isn't necessary, since all that the code is doing there is launching the container to grab a copy of the specific package versions that were installed and output `requirements.txt`.  So I can remove that there, as its not intended to perform any analyses.  

shm_size is set for analysis purposes in the compose yml now --- still to 2gb, although, for users with sufficient memory analysing larger cities, they may want/need to increase that:
https://github.com/global-healthy-liveable-cities/global-indicators/blob/931579fc4d375ac8dad25a532256ac84741f6745/docker-compose.yml#L6

Re: the batch file for building the docker image, I'll make a note to edit this later in the week.  If you think the setting is no longer required in the compose file, feel free to let me know.